### PR TITLE
Added Myself to the Contributers File and fixed link issue

### PR DESCRIPTION
**Pull request by whinis**
This fixes the problem reported in Issue 23. All links should have ../../ in front of them to fix linking issues.

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -4,4 +4,6 @@
 # here.  Please keep the list sorted by first names.
 
 Ed "Ortho Loess" Landamore
+Mark "Salacious Necrosis" Astley
 Regner "CCP FoxFour" Blok-Andersen
+John "Whinis/Equto" Turner
```
Comment (deadlybulb) [inline CONTRIBUTORS]: Think Regner wants to keep this sorted by first name.

File: docs/xmlapi/endpoints.md [hunk 2]
```diff
@@ -165,65 +165,233 @@ accessMask = 128
...
+        <tr>
+            <td></td>
+            <td>stationID</td>
+            <td nowrap>
+                <strong>int</strong>
+                <sup>
+                <a href="../../sde/tableref/#inventory-names">[1]</a>,
+                <a href="../../sde/tableref/#inventory-items">[2]</a>,
+                <a href="../../sde/tableref/#inventory-positions">[3]</a>,
+                <a href="../../sde/tableref/#inventory-unique-names">[4]</a>
```
Comment (deadlybulb) [inline docs/xmlapi/endpoints.md]: Nice!

File: CONTRIBUTORS (hunk 1 above)
Comment (whinis) [inline CONTRIBUTORS]: I probably should have read more, also wasn't sure how to put in my name as I was both whinis and equto. Neither is more prominent.

**Comment by Regner** [comment]
Any chance you can make this a PR for the ccpgames project?

**Pull request closed without merging**